### DISCUSSION TITLE
alist下载时弹出{ "error_code":31326, "error_msg":"anti hotlinking" }

### DISCUSSION THREAD

**Discussion by Gary414** [AlistGo/alist#4664] [Q&A]
用alist下载百度网盘上的东西时，弹出{ "error_code":31326, "error_msg":"anti hotlinking" } 。

https://github.com/alist-org/alist/assets/137972757/2a80b50f-25fb-4c4c-96d4-d36e7740836e

**Comment by 1273082756** [comment]
同问

**Comment by fastzk** [comment]
我也出现这个问题了，目前好几天没法解决，升级了最新版本，删除存储，重新添加，没有改善。
![1700708105996](https://github.com/alist-org/alist/assets/151803704/9f4f7e3c-4003-461d-8f76-3b7fa05faafc)

**Comment by 9-2-1** [comment]
可以将存储设置的“Web代理”和“WebDAV 本地代理”功能打开，这样下载文件会通过alist中转，alist应该会传递正确的校验数据。

坏处是如果你在服务器跑Alist，那么下载文件会二倍消耗Alist所在服务器的流量。Alist在本地则不受影响
